web/src/beta/components/ActionItem/index.stories.tsx

```diff
@@ -10,5 +10,9 @@ export default {
 type Story = StoryObj<typeof ActionItem>;
 
 export const Default: Story = {
-  render: () => <ActionItem title={"New Page"} icon={"square"} onClick={action("onClick")} />,
+  args: {
+    title: "New Page",
+    icon: "square",
+    onClick: action("onClick"),
```

Comment:
you don't need manually pass this function.
If the field name matches [this Regex](https://github.com/reearth/reearth/blob/main/web/.storybook/preview.tsx#L39), it will be automatically attached.